gravitee-apim-console-webui/src/portal/customization/top-bar/menu-link-edit/menu-link-edit.component.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, DestroyRef, inject, OnInit } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+import { ReactiveFormsModule, Validators, FormControl, FormGroup } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { GioSaveBarModule } from '@gravitee/ui-particles-angular';
+import { MatAnchor } from '@angular/material/button';
+import { MatIcon } from '@angular/material/icon';
+import { ActivatedRoute, RouterLink } from '@angular/router';
+import { tap } from 'rxjs/operators';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+import { PortalHeaderComponent } from '../../../components/header/portal-header.component';
+import { UiPortalMenuLinksService } from '../../../../services-ngx/ui-portal-menu-links.service';
+import { PortalMenuLink, toReadableMenuLinkType, UpdatePortalMenuLink } from '../../../../entities/management-api-v2';
+import { SnackBarService } from '../../../../services-ngx/snack-bar.service';
+
+@Component({
+  selector: 'menu-link-edit',
+  templateUrl: './menu-link-edit.component.html',
+  styleUrls: ['./menu-link-edit.component.scss'],
+  imports: [
+    CommonModule,
+    GioSaveBarModule,
+    MatCardModule,
+    MatFormFieldModule,
+    MatInputModule,
+    ReactiveFormsModule,
+    PortalHeaderComponent,
+    MatAnchor,
+    MatIcon,
+    RouterLink,
+  ],
+  standalone: true,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MenuLinkEditComponent implements OnInit {
+  private destroyRef: DestroyRef = inject(DestroyRef);
```

Comment:
Linting best practice is for private variables to be listed after public ones and before the constructor.